<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIPanelConfig</class>
 <widget class="QWidget" name="UIPanelConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>0</number>
     </property>
     <widget class="QListWidget" name="listWidget">
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="UIConfigSuffixEdit" name="configSuffixEdit" native="true"/>
       </item>
       <item>
        <widget class="UIConfigIgnoreFolder" name="ignoreFolder" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UIConfigSuffixEdit</class>
   <extends>QWidget</extends>
   <header>UIConfigSuffixEdit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UIConfigIgnoreFolder</class>
   <extends>QWidget</extends>
   <header>UIConfigIgnoreFolder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
